apps/scale.go

```diff
@@ -107,6 +109,13 @@ func Scale(app string, sync bool, types []string) error {
 	res, err := c.AppsScale(app, scaleParams)
 	if err != nil {
 		if !utils.IsPaymentRequiredAndFreeTrialExceededError(err) {
+			reqestFailedError, _ := errors.ErrgoRoot(err).(*http.RequestFailedError)
+
+			// In case of unprocessable, format and return an clear error
+			if reqestFailedError.Code == 422 {
+				return formatContainerTypesError(c, app, reqestFailedError)
```

Comment:
In case of error formatting the error message, I wouldn't return the error. I think it would be best to:
- Log the error
- Display the error message contained in `requestFailedError`. This error message is already a good error message. If we fail to enrich it with more informative information, we don't want to lose the source of the error message. Don't you think?